src/sentry/workflow_engine/models/data_source.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class DataSource(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    class Type(models.IntegerChoices):
+        SNUBA_QUERY_SUBSCRIPTION = 1
+
+    organization = FlexibleForeignKey("sentry.Organization")
+    query_id = BoundedBigIntegerField()
```

Comment:
I wasn't sure if we actually needed the other `query` column, anymore, since I think we were planning on avoiding json?

Reply:
i think we can figure that out when we get to issue alerts, the `query` here was meant to support the class id structure.